modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -1,36 +1,35 @@
 // Module included in the following assemblies:
 //
 // * applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc
+
 :_content-type: PROCEDURE
 [id="odc-creating-a-binding-connection-between-components_{context}"]
 = Creating a binding connection between components
 
-You can establish a binding connection with Operator-backed components.
+You can create a binding connection between components, as demonstrated in the following example that uses a PostgresSQL database service and a Spring PetClinic sample application.
 
-This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Spring PetClinic sample application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub*, and then install the Operator.
-The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.
+To create a binding connection with a service that is backed by the Crunchy Postgres for Kubernetes, add the Crunchy Postgres for Kubernetes Operator to the *OperatorHub*, and install the Operator. The Crunchy Postgres for Kubernetes Operator provides the PostgresCluster resource that you can use to deploy a PostgreSQL database and expose the binding information in secrets, config maps, status, and spec attributes.
```

Comment:
Dhruv, I'm aware that these lines have been in the file before this PR, but to me, this paragraph appears redundant for the following reasons:

The first sentence is a restatement of one of the prerequisites. It might not be the best use of the introduction to the procedure, which should summarize or explain the procedure, not focus on a prerequisite.

The second sentence is a description of an Operator, which is also listed in the prerequisites. I'd assume that if a reader decides to try this example, at the very least they will read the description of this Operator that is provided when installing it in the OperatorHub.

Reply:
@Dhruv-Soni11 and @max-cx This is the landing page for SBO on ODC from the SBO errata and we need [this old content](https://docs.openshift.com/container-platform/4.10/applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.html#creating-a-binding-connection-between-components) back as the abstract.

The old one gives a clear and general introduction and details to the user and reads well compared to the latest one.

Reply:
Keeping the old content.